airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py

```diff
@@ -95,41 +93,55 @@ def format_java_container(dagger_client: dagger.Client) -> dagger.Container:
             " spotless:check"
             " clean"
         ],
-        include=["**/*.java", "**/*.gradle"],
+        dir_to_format=java_code,
     )
 
 
-def format_js_container(dagger_client: dagger.Client) -> dagger.Container:
-    """Format yaml and json code via prettier."""
+def format_js_container(dagger_client: dagger.Client, js_code: dagger.Directory, prettier_version: str = "3.0.3") -> dagger.Container:
+    """Create a Node container with prettier installed with mounted code to format and a cache volume."""
     return build_container(
         dagger_client,
         base_image=NODE_IMAGE,
-        include=["**/*.yaml", "**/*.yml", "**.*/json", "package.json", "package-lock.json"],
-        install_commands=["npm install -g npm@10.1.0 prettier@3.0.3"],
+        dir_to_format=js_code,
+        install_commands=[f"npm install -g npm@10.1.0 prettier@{prettier_version}"],
+        cache_volume=dagger_client.cache_volume(slugify(f"prettier-{prettier_version}")),
```

Comment:
💅 Would still love if these were held in a `cache_key.py` file! But thats minor

Reply:
Done [b61b24e](https://github.com/airbytehq/airbyte/pull/33097/commits/b61b24e91cad43632b6d393388277e552f6f4599)